code/__DEFINES/items_clothing.dm

```diff
@@ -0,0 +1,190 @@
+#define slot_wear_suit_str	"slot_suit"
+#define slot_s_store_str    "slot_s_store"
+
+// Bitflags for clothing parts.
+#define HEAD        0x1
+#define FACE        0x2
+#define EYES        0x4
+#define EARS        0x8
+#define UPPER_TORSO 0x10
+#define LOWER_TORSO 0x20
+#define LEG_LEFT    0x40
+#define LEG_RIGHT   0x80
+#define LEGS        0xC0    //  LEG_LEFT | LEG_RIGHT
+#define ARM_LEFT    0x400
+#define ARM_RIGHT   0x800
+#define ARMS        0xC00   //  ARM_LEFT | ARM_RIGHT
+#define FULL_BODY   0xFFFF
+
+// Bitflags for the percentual amount of protection a piece of clothing which covers the body part offers.
+// Used with human/proc/get_heat_protection() and human/proc/get_cold_protection().
+// The values here should add up to 1, e.g., the head has 30% protection.
+#define THERMAL_PROTECTION_HEAD        0.3
+#define THERMAL_PROTECTION_UPPER_TORSO 0.15
+#define THERMAL_PROTECTION_LOWER_TORSO 0.15
+#define THERMAL_PROTECTION_LEG_LEFT    0.1
+#define THERMAL_PROTECTION_LEG_RIGHT   0.1
+#define THERMAL_PROTECTION_ARM_LEFT    0.1
+#define THERMAL_PROTECTION_ARM_RIGHT   0.1
+
+// Pressure limits.
+#define  HAZARD_HIGH_PRESSURE 550 // This determines at what pressure the ultra-high pressure red icon is displayed. (This one is set as a constant)
+#define WARNING_HIGH_PRESSURE 325 // This determines when the orange pressure icon is displayed (it is 0.7 * HAZARD_HIGH_PRESSURE)
+#define WARNING_LOW_PRESSURE  50  // This is when the gray low pressure icon is displayed. (it is 2.5 * HAZARD_LOW_PRESSURE)
+#define  HAZARD_LOW_PRESSURE  20  // This is when the black ultra-low pressure icon is displayed. (This one is set as a constant)
+
+#define TEMPERATURE_DAMAGE_COEFFICIENT  1.5 // This is used in handle_temperature_damage() for humans, and in reagents that affect body temperature. Temperature damage is multiplied by this amount.
+#define BODYTEMP_AUTORECOVERY_DIVISOR   12  // This is the divisor which handles how much of the temperature difference between the current body temperature and 310.15K (optimal temperature) humans auto-regenerate each tick. The higher the number, the slower the recovery. This is applied each tick, so long as the mob is alive.
+#define BODYTEMP_AUTORECOVERY_MINIMUM   1   // Minimum amount of kelvin moved toward 310.15K per tick. So long as abs(310.15 - bodytemp) is more than 50.
+#define BODYTEMP_COLD_DIVISOR           6   // Similar to the BODYTEMP_AUTORECOVERY_DIVISOR, but this is the divisor which is applied at the stage that follows autorecovery. This is the divisor which comes into play when the human's loc temperature is lower than their body temperature. Make it lower to lose bodytemp faster.
+#define BODYTEMP_HEAT_DIVISOR           6   // Similar to the BODYTEMP_AUTORECOVERY_DIVISOR, but this is the divisor which is applied at the stage that follows autorecovery. This is the divisor which comes into play when the human's loc temperature is higher than their body temperature. Make it lower to gain bodytemp faster.
```

Comment:
Superseded by and conflicts with the identical define in `code/__DEFINES/atmospherics.dm`.